src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1166,6 +1166,24 @@ public function pipesCodeProvider()
     }
 
     /**
+     * @dataProvider provideVariousIncrementals
+     */
+    public function testIncrementalOutputDoesNotRequiresAnotherCall($stream, $method) {
+        $process = new Process("php -r '\$n = 0; while (\$n < 3) { fwrite($stream, \$n, 1); \$n++; usleep(1000); }'", null, null, null, null);
```

Comment:
please use `self::$phpBin` instead of `php` direclty as done elsewhere to enhance compatibility of tests with more runtimes